### DISCUSSION TITLE
Mind Ar with personal image

### DISCUSSION THREAD

**Discussion by as20222** [hiukim/mind-ar-js#110] [Q&A]
Hi, I am implementing the basic example of mindAr but I want to use my own image which I have converted to .mind with the converter.
I get the following error when running the web, does anyone know a solution?

On the other hand, I want to know if the conversion to .mind can be done from the web's own code without using the converter

Thank you!

mindar-image.prod.js?v1642159871:2 Uncaught (in promise) RangeError: Extra 38464 byte(s) found at buffer[1]
    at e.createNoExtraBytesError (mindar-image.prod.js?v1642159871:2:16500)
    at e.decodeSingleSync (mindar-image.prod.js?v1642159871:2:16675)
    at Module.W (mindar-image.prod.js?v1642159871:2:24874)
    at Compiler.importData (mindar-image.prod.js?v1642159871:2:1134933)
    at Controller.addImageTargetsFromBuffer (mindar-image.prod.js?v1642159871:2:1136692)
    at mindar-image.prod.js?v1642159871:2:1136614
e.createNoExtraBytesError @ mindar-image.prod.js?v1642159871:2
e.decodeSingleSync @ mindar-image.prod.js?v1642159871:2
W @ mindar-image.prod.js?v1642159871:2
importData @ mindar-image.prod.js?v1642159871:2
addImageTargetsFromBuffer @ mindar-image.prod.js?v1642159871:2
(anonymous) @ mindar-image.prod.js?v1642159871:2
await in (anonymous) (async)
addImageTargets @ mindar-image.prod.js?v1642159871:2
_startAR @ mindar-image-aframe.prod.js?v1642159871:1
(anonymous) @ mindar-image-aframe.prod.js?v1642159871:1

**Comment by alkoumi** [comment]
Can I use the compiler on my own site and how?

**Comment by uptownben** [comment]
I got that error when the `imageTargetSrc:` was wrong or the code could not find the `.mind` file. When you run the web app, open your Developer Tool (F12 on Windows), and check the Network tab. Check if the `.mind` file is loaded correctly.
